<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatmapDock</class>
 <widget class="QWidget" name="HeatmapDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>677</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="11" column="3">
        <widget class="QPushButton" name="bColorMap">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QSpinBox" name="sbyNumberBins"/>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacerST_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="19" column="0">
        <spacer name="verticalSpacerGeneral">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lSameNumberBins">
         <property name="text">
          <string>Same Number</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="lPlotRange">
         <property name="text">
          <string>Plot Range:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2" colspan="2">
        <widget class="NumberSpinBox" name="sbLimitsMax"/>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="cbSameNumberBins">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lyNumberBins">
         <property name="text">
          <string>Number Bins Y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lXColumn">
         <property name="text">
          <string>X Column:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lColorMap">
         <property name="text">
          <string>Color Map:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="20" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2" colspan="2">
        <widget class="NumberSpinBox" name="sbLimitsMin"/>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QSpinBox" name="sbxNumberBins"/>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="lColorMapPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lDataSource">
         <property name="text">
          <string>Data Source:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="ResizableTextEdit" name="teComment"/>
       </item>
       <item row="18" column="2" colspan="2">
        <widget class="QComboBox" name="cbPlotRanges"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lMatrix">
         <property name="text">
          <string>Matrix:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacerST">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Automatic</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="hsName">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QComboBox" name="cbDataSource"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lxNumberBins">
         <property name="text">
          <string>Number Bins X</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QCheckBox" name="cbAutomaticLimits">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lYColumn">
         <property name="text">
          <string>Y Column:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <layout class="QHBoxLayout" name="hBoxMatrix"/>
       </item>
       <item row="5" column="2" colspan="2">
        <layout class="QHBoxLayout" name="hBoxXColumn"/>
       </item>
       <item row="6" column="2" colspan="2">
        <layout class="QHBoxLayout" name="hBoxYColumn"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
